src/shared/ranking.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * This function get data from localStorage by category name.
+ * @param {string} category
+ * Function return [{username: string, points: number}]
+ */
+export function getRankingByCategory(category) {
+  const rankingStorage = window.localStorage.getItem(category);
+  if (rankingStorage == null) {
+    return false;
```

Comment:
Chyba lepiej zwrócić pustą tablicę, później przy korzystaniu z tego nie będzie problemów jeśli ktoś zrobi np. `.map()` bez sprawdzenia czy wynik nie był `false`.

Reply:
tylko rankingStorage to w sumie nie jest tablica, to cięzko będzie zwrócic pusty element.

Reply:
```suggestion
    return { category, userData: [] };
```

Miałem na myśli np. coś takiego.

Reply:
dodane, wygląda, że działa.